easybuild/easyconfigs/s/SWIG/SWIG-3.0.10-intel-2016b-Python-2.7.12.eb

```diff
@@ -14,7 +14,7 @@ sources = [SOURCELOWER_TAR_GZ]
 
 dependencies = [
     ('Python', '2.7.12'),
-    ('PCRE', '8.38'),
+    ('PCRE', '8.39'),
```

Comment:
really not a big fan of doing this, since this may cause problems for people using this easyconfig already as a dep elsewhere...

The only way out I can think of is to create an easyconfig for SWGI 3.0.10 with `-PCRE-8.39` as extra part to the versionsuffix...